src/LatSpec.jl

```diff
@@ -1,14 +1,15 @@
 module LatSpec
 
+import LsqFit: curve_fit
+import Statistics: mean
+using Plots
+
```

Comment:
I've read somewhere that there is a suggested order for structuring a module-file. Do you know anything about it and if so do we already follow this?

Reply:
To be honest I don't care. Put them in any order you want ;)